### ISSUE TITLE
Missing version number in index page

### ISSUE THREAD

**Issue by nicfv** [tgreyuk/typedoc-plugin-markdown#553]
When using `--includeVersion`, the version number is missing from the index page but shown on all other pages.

**Comment by nicfv** [comment]
I should mention this is an issue with the breadcrumbs:

In README.md:
```md
package-name / [Exports](modules.md)
```

(Should be `package-name - v1.0.0`)

In modules.md
```md
[package-name - v1.0.0](README.md) / Exports
```

In one of my class files:
```md
[package-name - v1.0.0](../README.md) / [Exports](../modules.md) / [Geometry/Length](../modules/Geometry_Length.md) / Length
```

**Comment by tgreyuk** [comment]
Fix in `typedoc-plugin-markdown@4.0.0-next.45`
